src/cmd/create.go

```diff
@@ -123,44 +139,60 @@ func create(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	var release string
-	if createFlags.release != "" {
-		var err error
-		release, err = utils.ParseRelease(createFlags.release)
-		if err != nil {
-			err := utils.CreateErrorInvalidRelease(executableBase)
-			return err
+	if cmd.Flag("distro").Changed {
+		if !toolbox.IsSystemSupported(createFlags.distro) {
+			var builder strings.Builder
+			fmt.Fprintf(&builder, "Toolbox does not support using distribution %s for toolboxes\n", createFlags.distro)
+			fmt.Fprintf(&builder, "Run '%s create --help' to see a list of supported distributions.", executableBase)
+
+			errMsg := builder.String()
+			return errors.New(errMsg)
 		}
-	}
 
-	container, image, release, err := utils.ResolveContainerAndImageNames(container,
-		createFlags.image,
-		release)
-	if err != nil {
-		return err
+		image, _ = toolbox.GetImageForSystem(createFlags.distro)
+	} else {
+		if toolbox.IsHostSystemSupported() {
+			hostID, err := utils.GetHostID()
+			if err != nil {
+				return fmt.Errorf("There was an error while getting host's ID: %v", err)
```

Comment:
It's [better](https://blog.golang.org/go1.13-errors) to use `%w` when wrapping errors.